working-with-collections/main.js

```diff
@@ -0,0 +1,67 @@
+/* global _ */
+/* eslint-disable no-console */
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+
+setTimeout(() => { console.log('Type "play()" in console for A New Game'); }, 500);
+
+const cardsSuit = ['clubs', 'diamonds', 'hearts', 'spades'];
+const cardsRank = ['A', 2, 3, 4, 5, 6, 7, 8, 9, 10, 'J', 'Q', 'K'];
+const cardsData = [];
+const player = ['Annie', 'Tim D.', 'Cody', 'Uzair'];
+const cardsPerHand = 2;
+
+function createCards(suit, rank) {
```

Comment:
I recommend that you create everything for the cards in this function instead of global variables. It will make the code much easier to read.

Reply:
I moved these to play function